README.md

```diff
@@ -1,19 +1,97 @@
-# Homebrew Test Bot
+Homebrew Test Bot
+=================
```

Comment:
Please don't make changes like this, it makes the overall changes hard to follow based on your syntax preferences.